package.json

```
@@ -13,8 +13,8 @@
   },
   "scripts": {
     "prebuild": "run-s test",
-    "build": "microbundle-crl --no-compress --format modern,cjs --css-modules false",
-    "start": "microbundle-crl watch --no-compress --format modern,cjs --css-modules false",
+    "build": "rm -rf dist && microbundle-crl --no-compress --format modern,cjs --css-modules false",
+    "start": "rm -rf dist && microbundle-crl watch --no-compress --format modern,cjs --css-modules false",
```

This is unrelated to your PR but if we are already discussing this stuff, then...

```suggestion
    "start": "rm -rf dist && microbundle-crl watch --no-compress --format modern --css-modules false",
```

In watch mode do we really need to create a cjs bundle?

---

+1
cjs removed